### DISCUSSION TITLE
deployments: add a description to manual deployments

### DISCUSSION THREAD

**Discussion by gustavovalverde** [ZcashFoundation/zebra#3583] [Ideas]
## Motivation
When running a workflow manually, using `workflow_dispatch`, it's commonly done with some reasoning behind it. But the machines on GCP don't have this reasoning exposed anywhere, and neither on the GitHub Action.

### Designs

Add a field to add reasoning for doing a specific manual deployment, and keep this information available in GitHub Actions and in GCP.

**Comment by dconnolly** [comment]
I know I would will several of these with "test" :P

**Comment by conradoplg** [comment]
This would be useful, there's a bunch of deployments that I have no idea if we can delete or not :laughing: 

Is there anything in particular about this that you'd like feedback on?

**Comment by mpguerra** [comment]
closing as stale
